welcome-checklist.md

```diff
@@ -26,11 +26,21 @@ Before we begin, we ask that you complete this list to ensure that you have acce
 
 * In the terminal, type `python --version`. You should see the version of your Python program listed.
     * Mac and Linux: If you get an error or your version does not indicate "Anaconda", put your red sticky note on top of your laptop.
-    * Windows: If you receive an error message about python not being found, run the following command:
-      ```
-      echo  ". \$USERPROFILE/AppData/Local/Continuum/anaconda3/Scripts/activate" >> ~/.bashrc
-      ```
-      then open a **new** terminal and try typing `python --version` again.
+    * Windows: If you receive an error message about python not being found, do the following:
+      * Open the Anaconda Prompt and enter the following to find the path of the Anaconda3 directory:
+        ```
+        where python
+        ```
+        The output should show a path similar to `C:\Users\Username\Anaconda3\python.exe`.
+      * Convert that path by doing the following:
+        * Replace all instances of `\` with `/`
+        * Replace `C:` with `/c`
+        * Remove the `\python.exe` part at the end
+      * Go back to the terminal and run the following command, replacing `[Anaconda3 location]` with the path you generated in the previous step.
```

Comment:
I liked it how you previously had "for example, if you got `C:\Users\Username\Anaconda3\python.exe` from the previous step, you will use `/c/Users/Username/Anaconda3` for the next step", can we work that back in?

Reply:
Definitely. I'm also wondering if maybe we should have them replace the `\python.exe` with `/Scripts/activate` here rather than adding that in the next step.

Reply:
Yeah maybe it should be "Replace `python.exe` with `Scripts/activate`"